signal.go

```diff
@@ -32,16 +33,17 @@ func mainInterruptHandler() {
 
 	for {
 		select {
-		case <-interruptChannel:
+		case sig := <-interruptChannel:
 			// Ignore more than one shutdown signal.
 			if isShutdown {
-				btcdLog.Infof("Received SIGINT (Ctrl+C).  " +
-					"Already shutting down...")
+				btcdLog.Infof("Received signal (%s).  "+
+					"Already shutting down...", sig.String())
```

Comment:
Since sig is already a `fmt.Stringer`, there is no need to explicitly call `.String()`.

The same is true for the next one.